component/main.jsonnet

```diff
@@ -34,14 +34,27 @@ local ns_patch =
     }
   );
 
+local transformRelabelConfigs(remoteWriteConfig) = if std.member(std.objectFields(remoteWriteConfig), 'writeRelabelConfigs')
+then remoteWriteConfig {
+  writeRelabelConfigs: std.map(
+    function(wrlc) { [key]: wrlc[key] for key in std.objectFields(wrlc) if key != 'timeseries' } {
+      [if std.member(std.objectFields(wrlc), 'timeseries') && std.length(wrlc.timeseries) > 0
+      then 'regex'
+      else null]: std.format('(%s)', std.join('|', wrlc.timeseries)),
```

Comment:
`else null` is implied when you just use `if cond then expr`, but it doesn't hurt to have it either.